cypress/e2e/landing.test.ts

```diff
@@ -26,7 +26,15 @@ describe('Landing Page', () => {
 
   it('allows navigation to pool', () => {
     cy.viewport(2000, 1600)
+    cy.visit('/swap')
     cy.get(getTestSelector('pool-nav-link')).first().click()
     cy.url().should('include', '/pools')
   })
+
+  it('allows navigation to pool on mobile', () => {
+    cy.viewport(375, 667)
```

Comment:
Where are these numbers from? Can you leave a comment, and maybe make them a constant?

```
const DESKTOP_DIMENSIONS = [2000, 1600]
const MOBILE_DIMENSIONS = [375, 667]
...
cy.viewport(...DESKTOP_DIMENSIONS)
...
cy.viewport(...MOBILE_DIMENSIONS)
```